### PR TITLE
feat: Add update services and rename packages

### PR DESCRIPTION
**Pull request by joshua-stone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,45 @@
-[![build-ublue](https://github.com/ublue-os/udev-rules/actions/workflows/build.yml/badge.svg)](https://github.com/ublue-os/udev-rules/actions/workflows/build.yml)
+[![build-ublue](https://github.com/ublue-os/confif/actions/workflows/build.yml/badge.svg)](https://github.com/ublue-os/config/actions/workflows/build.yml)
```
Comment (castrojo) [inline README.md:1]: confif -> `config`

File: README.md [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
-A layer for adding extra udev rules to your image. Use this for better hardware support!
+A layer for adding enhancements to your image. Use these for better hardware support and automatic updates!
 
 # Usage
 
 Add this to your Containerfile to copy the rules over:
 
-    COPY --from=ghcr.io/ublue-os/udev-rules:latest /ublue-os-udev-rules /
+    COPY --from=ghcr.io/ublue-os/udev-rules:latest /files/ublue-os-udev-rules /
+    COPY --from=ghcr.io/ublue-os/udev-rules:latest /files/ublue-os-update-services /
```
Comment (castrojo) [inline README.md:12]: Shouldn't these be `config` instead of udev-rules?

File: README.md [hunk 3]
```diff
@@ -1,35 +1,45 @@
...
     
 Or if you prefer to install via an RPM package:
 
-    COPY --from=ghcr.io/ublue-os/udev-rules:latest /ublue-os-udev-rules.noarch.rpm /
+    COPY --from=ghcr.io/ublue-os/udev-rules:latest /rpms/ublue-os-udev-rules.noarch.rpm /
+    COPY --from=ghcr.io/ublue-os/udev-rules:latest /rpmm/ublue-os-update-services.noarch.rpm /
```
Comment (castrojo) [inline README.md:17]: Shouldn't these be `config` instead of udev-rules?

**Review by castrojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joshua-stone) [reply README.md:1]: Good catch!

**Review by castrojo — APPROVED** [review]

**Pull request merged**